### PR TITLE
Alchemix Next tokens

### PR DESCRIPTION
**Pull request by Hesnicewithit**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

**Comment by LayneHaber** [comment]
When I simulate the whitelisting transaction, it fails at minting:

1. alUSD arb:
https://dashboard.tenderly.co/lhaber/project/simulator/d0031007-8894-4fb6-9c15-2e2ab8a2837e

2. alUSD opt:
https://dashboard.tenderly.co/lhaber/project/simulator/d91c495d-7197-4719-b5b9-86de53ab1403

This is a preliminary step in our whitelisting to ensure the bridge can mint/burn on the representation. alETH seems connext isn’t whitelisted on arbitrum-goerli, but the others are. What’s odd is it seems to call mint on the alAsset, which is failing. Can’t dig into why because the source isn’t available

File: packages/deployments/contracts/src/cli/init/config/testnet/production.ts [hunk 1]
```diff
@@ -94,6 +94,46 @@ export const TESTNET_PRODUCTION_INIT_CONFIG: InitConfig = {
         },
       },
     },
+    {
+      name: "nextAlUSD",
```
Comment (LayneHaber) [inline packages/deployments/contracts/src/cli/init/config/testnet/production.ts:98]: should match the canonical symbol
```suggestion
      name: "alUSD",
```

File: packages/deployments/contracts/src/cli/init/config/testnet/production.ts [hunk 2]
```diff
@@ -94,6 +94,46 @@ export const TESTNET_PRODUCTION_INIT_CONFIG: InitConfig = {
...
+        },
+        /// OPTIMISM-GOERLI
+        "1735356532": {
+          local: "0x6e56Cb72f4197b8155Cf8e1efaEFB682BC9ae5d1",
+          adopted: "0x6e56Cb72f4197b8155Cf8e1efaEFB682BC9ae5d1",
+        },
+      },
+    },
+    {
+      name: "nextAlETH",
```
Comment (LayneHaber) [inline packages/deployments/contracts/src/cli/init/config/testnet/production.ts:118]: should match the canonical symbol
```suggestion
      name: "alETH",
```

**Review by LayneHaber — CHANGES_REQUESTED** [review]
see above re: simulations

**Review by rhlsthrm — APPROVED** [review]

**Review by sanchaymittal — APPROVED** [review]

**Pull request merged**